### ISSUE TITLE
[Suggestion] Enable blockstate culling on scaffoldings

### ISSUE THREAD

**Issue by Felix14-v2** [FxMorin/MoreCulling#154]
Scaffolding has one of the most expensive block models in the entire game, and MoreCulling can optimize it a lot. In the worst case, there are 16 useless faces per block that can be culled
![image](https://user-images.githubusercontent.com/75726196/229175062-855e29bc-5d96-4e09-b7a1-9a1cbbe57fd8.png)
![image](https://user-images.githubusercontent.com/75726196/229176657-1b0e409e-80ca-4240-803d-bffcfb01382c.png)

**Comment by fxmorin** [comment]
Well opengl is already applying block face culling to it. So not really, however I believe it should be culling but is unable to do to the fact it does not have collisions

**Comment by FxMorin** [comment]
Turns out it was not due to the collision, as I fixed that.
I've determined the cause. It believes that the legs have transparent pixels in them. So it doesn't attempt to cull it.
I implemented directional texture transparency in v0.13.0, however that somehow didn't seem to work for scaffolding.

So I believe the cause is due to the side scaffolding texture having transparency between the legs in the texture. I'll see if I have enough time to write code to take that into account.
